patches/remove-useless-ui.patch

```diff
@@ -0,0 +1,270 @@
+diff --git a/chrome/browser/resources/settings/people_page/people_page.ts b/chrome/browser/resources/settings/people_page/people_page.ts
+index 3ec954ed874f3..36e08094951bc 100644
+--- a/chrome/browser/resources/settings/people_page/people_page.ts
++++ b/chrome/browser/resources/settings/people_page/people_page.ts
+@@ -354,12 +354,7 @@ export class SettingsPeoplePageElement extends SettingsPeoplePageElementBase {
+ 
+   // <if expr="not chromeos_ash">
+   private computeShouldShowGoogleAccount_(): boolean {
+-    if (this.storedAccounts === undefined || this.syncStatus === undefined) {
+-      return false;
+-    }
+-
+-    return (this.storedAccounts!.length > 0 || this.isSyncing_()) &&
+-        !this.syncStatus!.hasError;
++    return false;
+   }
+   // </if>
+ 
+@@ -412,15 +407,7 @@ export class SettingsPeoplePageElement extends SettingsPeoplePageElementBase {
+   }
+ 
+   private shouldShowSyncAccountControl_(): boolean {
+-    // <if expr="chromeos_ash">
+     return false;
+-    // </if>
+-    // <if expr="not chromeos_ash">
+-    if (this.syncStatus === undefined) {
+-      return false;
+-    }
+-    return !!this.syncStatus!.syncSystemEnabled && this.signinAllowed_;
+-    // </if>
+   }
+ 
+   /**
+diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.ts b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
+index 00318322c8147..409f855a3a2b1 100644
+--- a/chrome/browser/resources/settings/privacy_page/privacy_page.ts
++++ b/chrome/browser/resources/settings/privacy_page/privacy_page.ts
+@@ -417,9 +417,7 @@ export class SettingsPrivacyPageElement extends SettingsPrivacyPageElementBase {
+   override currentRouteChanged() {
+     this.showClearBrowsingDataDialog_ =
+         Router.getInstance().getCurrentRoute() === routes.CLEAR_BROWSER_DATA;
+-    this.showPrivacyGuideDialog_ =
+-        Router.getInstance().getCurrentRoute() === routes.PRIVACY_GUIDE &&
+-        this.isPrivacyGuideAvailable;
++    this.showPrivacyGuideDialog_ = false;
+ 
+     // Only record the metrics when the user navigates to the notification
+     // settings page that shows the entry point.
+@@ -608,8 +606,7 @@ export class SettingsPrivacyPageElement extends SettingsPrivacyPageElementBase {
+   }
+ 
+   private shouldShowAdPrivacy_(): boolean {
+-    return !this.isPrivacySandboxRestricted_ ||
+-        this.isPrivacySandboxRestrictedNoticeEnabled_;
```

Comment:
this will work but it would be even better if we could tie this to the privacysandbox feature disabled by default here:

https://github.com/secureblue/hardened-chromium/blob/master/vanadium_patches/0046-Disable-newer-privacy-sandbox-features-by-default.patch

That way we can control it from one place

Reply:
Oh, this is a frontend file. Are feature flags exposed in the frontnend?

Reply:
Ah okay, that's a shame. Nevermind then.